packages/client/src/schema/files.ts

```diff
@@ -20,56 +20,56 @@ export class XataFile {
   /**
    * Name of the file.
    */
-  public name: string;
+  public name?: string;
```

Comment:
All of these being optional feels like kind of a weird API for the user. Maybe we should at least add JSDocs for the defaults if not specified (if there are any)?

Reply:
Yeah, it's true that it feels weird. However that's what happens with any column right now. We could remove the undefined bit in column selection. I'll open a ticket and once I clean-up the `object notation` deprecated paths I'll attempt to improve this too.